examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -91,6 +91,11 @@ func (m *Memeland) GetPostsInRange(startTimestamp, endTimestamp int64, page, pag
 		panic("page size cannot be larger than 10")
 	}
 
+	// Need to pass in a sort parameter
+	if sortBy == "" {
+		panic("sortBy cannot be empty")
```

Comment:
Please don't use the actual param name in the panic message

Reply:
Fixed: [015902f](https://github.com/gnolang/gno/pull/1817/commits/015902f694b808371b3cfc3494a5423c465b5cfa)